cli-plugins/manager/manager.go

```diff
@@ -114,13 +113,21 @@ func listPluginCandidates(dirs []string) map[string][]string {
 	return result
 }
 
+// ConfigProvider defines an interface for providing the CLI config.
+type ConfigProvider interface {
+	ConfigFile() *configfile.ConfigFile
+}
```

Comment:
Actually now starting to consider if we should move this interface into `cli/config` or `cli/config/configfile`, as there's quite some places where all we're looking for is the Config 🤔